### PR TITLE
feat: videolisting page added

### PR DESCRIPTION
**Pull request by sharmayatendra**
# Changes

- Video listing page added

image

### CODE REVIEW

File: src/components/Nav/Nav.jsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+<header className="header">
+  <div className="header-left-icons">
+    
+    <i class="material-icons-outlined video-icon">
+    videocam
+    </i>
+    <Link to="/">
+    <p className="header-heading">AMAZE-PLAYER</p>
+    </Link>
+    {/* <input type="search" className='header-search' placeholder='Search'/> */}
```
Comment (dusariudaykumar) [inline src/components/Nav/Nav.jsx:16]: One suggestion remove unused code  before pushing the code

**Review by dusariudaykumar — APPROVED** [review]
- Code looks good to me
- The UI looks good
- The class naming convention is on point. Good work!

File: src/App.css [hunk 2]
```diff
@@ -1,4 +1,4 @@
-.App {
+/* .App {
```
Comment (tushargoyal812) [inline src/App.css:1]: remove it not needed

File: src/components/Nav/Nav.css [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+}
+.header-search {
+    padding: 0.75rem 1rem;
+    width: 30%;
+    background-color: rgb(20, 20, 20);
+    box-shadow: inset 2px 2px #0e0d0d;
+    caret-color: var(--color-white);
+    border: 1px solid black;
+    color: var(--color-white);
+    /* margin-left: 6rem; */
```
Comment (tushargoyal812) [inline src/components/Nav/Nav.css:53]: remove comments

File: src/components/Sidebar/Sidebar.css [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+}
+
+.list-item {
+    font-size: 1.2rem;
+}
+
+/* .list:hover {
+    background-color: rgb(204, 200, 200);
+    border-radius: 4px;
+} */
```
Comment (tushargoyal812) [inline src/components/Sidebar/Sidebar.css:42]: also here

File: src/index.css [hunk 5]
```diff
@@ -1,4 +1,4 @@
-body {
+/* body {
   margin: 0;
   font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
     "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
```
Comment (tushargoyal812) [inline src/index.css]: remove it if not needed

File: src/pages/VideoListing/VideoListing.jsx [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+import { Nav , Sidebar , VideoCard } from '../../components/index'
+
+function VideoListing() {
+  const [videos,setVideos] = useState([])
+
+  useEffect(() => {
+    (async function fetchVideos() {
+      try {
+        const resp = await axios.get("/api/videos")
+        console.log(resp);
```
Comment (tushargoyal812) [inline src/pages/VideoListing/VideoListing.jsx]: remove console

**Review by tushargoyal812 — COMMENTED** [review]
rest looks good to me

File: src/pages/VideoListing/VideoListing.jsx (hunk 6 above)
Comment (sharmayatendra) [reply src/pages/VideoListing/VideoListing.jsx]: will do

File: src/index.css (hunk 5 above)
Comment (sharmayatendra) [reply src/index.css]: ok will remove

**Pull request merged**